assets/messages.yml

```
@@ -510,6 +510,29 @@ messages:
 
         If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.
 
+        %quick_links%
+      fillname:
+        - Enter parent issue ID
+  duplicate-forward:
+    - project:
+        - bds
+        - mc
+        - mcapi
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Duplicate (forward)
+      shortcut: fodup
+      message: |-
+        *Thank you for your report!*
+        We're resolving and linking this issue forward as a *duplicate* of *%s%*, as that ticket contains more detailed information and/or has already been triaged by Mojang.
```

```suggestion
        We're resolving and linking this ticket forward as a *duplicate* of *%s%*, as that ticket contains more detailed information and/or has already been triaged by Mojang.
```
"Ticket" might be more correct, but I am not sure if it is understandable enough for the users.
Note that the other duplicate messages are using "issue" with the meaning of "bug".